siliconcompiler/remote/client.py

```diff
@@ -676,7 +676,7 @@ def __ensure_run_loop_information(self):
                 "imported": done,
                 "fetched": done
             }
-            self.__node_information[f'{step}{index}'] = node_info
+            self.__node_information[f'{step}/{index}'] = node_info
```

Comment:
this requires some changes on our server, so I think this needs to stay `f'{step}{index}'`